solidity/dashboard/src/css/card.less

```diff
@@ -0,0 +1,6 @@
+.card-root {
+  border: 1px solid black;
```

Comment:
As I can see on Figma the `border` prop is unnecessary here.

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/2216/commits/bd7dbb50d3b769a27468fe3e1e2f252924a8ee04